Get rid of accent marks

victorlopezalonso opened the pull request:
Get rid of accent marks on test folders to avoid problems when using composer install on unix based systems

victorlopezalonso committed:
Get rid of accent marks

AddonPayments commented:
Good morning Víctor López.

We have taken into account the information you have provided for this new version of the SDK where we have modified part of the documentation and added the updated JavaScript library, from now on accents will not be displayed in the folders.

Best regards,
Víctor Cantera.

victorlopezalonso commented:
> Good morning Víctor López.
> 
> We have taken into account the information you have provided for this new version of the SDK where we have modified part of the documentation and added the updated JavaScript library, from now on accents will not be displayed in the folders.
> 
> Best regards,
> Víctor Cantera.

Thank you!
